pmda/rdf.py

```diff
@@ -164,7 +164,7 @@ def _conclude(self, ):
     @staticmethod
     def _reduce(res, result_single_frame):
         """ 'add' action for an accumulator"""
-        if res == []:
+        if isinstance(res, list) and len(res) == 0:
```

Comment:
@VOD555 Can you have a look. Your old version triggered a deprecation warning when `res` is a numpy array.